### PR TITLE
Added notes for remote hosts on clusters.

### PR DESCRIPTION
**Pull request by rubenvp8510**
I'm not sure if it's clear enough Also, are we going to add something about how to change replication factor ?

@theute

### CODE REVIEW

File: mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc [hunk 1]
```diff
@@ -11,6 +11,9 @@ docker run -d -e CASSANDRA_SEEDS=mwmanager-cassandra -d \
               --link mwmanager-cassandra  \
               brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss/cassandra:latest
 ----
+NOTE: if the cluster needs to run on a separate box, the option -e CASSANDRA_BROADCAST_ADDRESS=<public-ip> must be specified on each node
```
Comment (theute) [inline mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc]: "on a separate host"

File: mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc [hunk 2]
```diff
@@ -11,6 +11,9 @@ docker run -d -e CASSANDRA_SEEDS=mwmanager-cassandra -d \
...
+that is launched, where public-ip is the IP of the machine on the network, is not possible to combine the approach of having
+multiple nodes on the same box and nodes on a remote box for one cluster.
```
Comment (theute) [inline mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc]: s/box/host

File: mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc [hunk 3]
```diff
@@ -54,6 +57,9 @@ This process will move data to the other nodes, and replicate the appropriate da
   docker exec -it my_container_id /usr/bin/nodetool decommission
 ----
 
+NOTE: If the node is running on another machine, you need to execute this command on the that machine.
```
Comment (theute) [inline mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc]: Which command ?

File: mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc [hunk 4]
```diff
@@ -110,6 +116,10 @@ docker run -d -e JVM_OPTS="-Dcassandra.replace_address=<node_replaced_ip>" \
            brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss/cassandra:latest
 
 ----
+
+NOTE: If the node is running on another machine, you need to execute this command on the that machine and must be specify
```
Comment (theute) [inline mw_manager_installation_guide/topics/cassandra_cluster_setup.adoc]: Let's use 'host' or 'machine'  everywhere (box sounds bad)'

**Comment by rubenvp8510** [comment]
Fixes some little issues in the text.

**Pull request merged**